README.md

```diff
@@ -35,18 +37,22 @@ This shopping list app is a small single page web application consisting of an H
 
 So what sets this app apart? Its Offline First architecture. The Offline First approach plans for the most constrained network environment first, enabling a great user experience even while the device is offline or has only an intermittent connection, and providing progressive enhancement as network conditions improve. This design also makes the app incredibly performant (fast!) on the best of networks. 
 
+<!--Adjust this paragraph if your implementation is NOT a PWA. -->
```

Comment:
We can probably remove these comments.

Reply:
Ignore this comment about removing comments. I had misunderstood what step in the process this was.